open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -715,34 +793,36 @@ def check_hackathon_online(self, hackathon):
 
         return ok(can_online)
 
+    # TODO: some information is omitted and needed to be added
     def __get_hackathon_detail(self, hackathon, user=None):
         """Return hackathon info as well as its details including configs, stat, organizers, like if user logon"""
         detail = hackathon.dic()
 
-        detail["config"] = self.__get_hackathon_configs(hackathon)
-        detail["stat"] = self.get_hackathon_stat(hackathon)
-        detail["tag"] = self.get_hackathon_tags(hackathon)
-        detail["organizer"] = self.__get_hackathon_organizers(hackathon)
+        # detail["config"] = self.__get_hackathon_configs(hackathon)
+        # detail["stat"] = self.get_hackathon_stat(hackathon)
+        # detail["tag"] = self.get_hackathon_tags(hackathon)
+        # detail["organizer"] = self.__get_hackathon_organizers(hackathon)
+        detail["stat"] = {"register": 5}
 
         if user:
             detail["user"] = self.user_manager.user_display_info(user)
-            detail["user"]["is_admin"] = self.admin_manager.is_hackathon_admin(hackathon.id, user.id)
-
-            asset = self.db.find_all_objects_by(UserHackathonAsset, user_id=user.id, hackathon_id=hackathon.id)
-            if asset:
-                detail["asset"] = [o.dic() for o in asset]
-
-            like = self.db.find_first_object_by(HackathonLike, user_id=user.id, hackathon_id=hackathon.id)
-            if like:
-                detail["like"] = like.dic()
-
-            register = self.register_manager.get_registration_by_user_and_hackathon(user.id, hackathon.id)
-            if register:
-                detail["registration"] = register.dic()
-
-            team_rel = self.db.find_first_object_by(UserTeamRel, user_id=user.id, hackathon_id=hackathon.id)
-            if team_rel:
-                detail["team"] = team_rel.team.dic()
+            # detail["user"]["is_admin"] = self.admin_manager.is_hackathon_admin(hackathon.id, user.id)
```

Comment:
Please add `todo` here: we need to review those items one by one to decide the API output